docs/SENTENCES.md

```diff
@@ -34,8 +42,26 @@ Feel free to set up this QA however makes most sense for you, but here's a [samp
 
 Once the review is complete, submit a pull request with the # of sentences submitted, a link to the manual QA results, and the % error rate. Here's [an example PR](https://github.com/mozilla/common-voice/pull/2873). Please make sure the sentences are in a plain `.txt` file with one sentence per line.
 
-## Flagging problematic sentences
+## QA that applies (or not) to the different imputs and outputs
+As a reminder, here is listed the different automated QA applied, depending of the process.
+* [cleanup](https://github.com/common-voice/sentence-collector/tree/main/server/lib/cleanup) and [validation](https://github.com/common-voice/sentence-collector/tree/main/server/lib/validation) rules for Sentence Collector, 
+* [language](https://github.com/common-voice/cv-sentence-extractor#using-language-rules) rules for Sentence Extractor, 
+* [preprocessor](https://github.com/common-voice/CorporaCreator/tree/master/src/corporacreator/preprocessors) for CorporaCreator ([CorporaCreator](https://github.com/common-voice/CorporaCreator) is the Common Voice extractor that, linking text and recorded voice samples, make them available for [SST](https://en.wikipedia.org/wiki/Speech_recognition) engines' training).
```

Comment:
I'm not sure about this CorporaCreator description, so I'll let others review this.

Reply:
CorporaCreator creates the default train, dev and test splits from validated recordings, taking a line of diversity rules and statistical correctness into account. These splits are used in training models for STT (Speech to Text).

PS: AFAIK, the preprocessors existing there are last minute correcting rules, as some (sometimes most) of the text-corpora are coming from bulk-submission PRs...

Reply:
This might help about inner workings (the second part of the post):
https://discourse.mozilla.org/t/inadequate-documentation/105415/2

Reply:
Different clarifications are in progress, both on README files ([here](https://github.com/common-voice/common-voice/pull/3811)) and on an overview of process ([here](https://discourse.mozilla.org/t/trying-to-create-a-common-voice-overview-for-newbies-svg-drawin-style/105771))...

I think it resolve the conversation